apps/code-examples/src/app/features/modals.module.ts

```diff
@@ -17,23 +17,30 @@ const routes: Routes = [
       ).then((c) => c.DemoComponent),
   },
   {
-    path: 'modal',
+    path: 'modal-with-controller',
```

Comment:
The main page of this app should probably be updated for the existing "Modal" link to not be broken and to have the new demo, correct?